ui/src/widgets/ui-gauge/UIGauge.vue

```diff
@@ -1,8 +1,6 @@
 <template>
-    <ui-gauge-tile v-if="props.gtype === 'gauge-tile'" :id="id" :props="props" :value="value" />
```

Comment:
replaced conditional gauge type from dynamic component except 3/4 and half as doesn't support for dynamic component

Reply:
all gauge types seem to be reacting to dynamic prop changes. What do you mean?

Reply:
I took all three "ui-gauge-battery", "ui-gauge-tank" and "ui-gauge-dial" into one `dynamic component Vue inbuilt element` just to reduce few lines of codes and to avoid `v-else-if` as `component` element handles them naturally but couldn't include `ui-gauge-dial` to that as it looses reactivity and didn't get rendered properly